test/unittest/tests/utils.h

```diff
@@ -13,6 +13,8 @@
 #define DB_HOME "test_db"
 
 namespace utils {
+#define BREAK utils::break_here(__FILE__, __func__, __LINE__)
+void break_here(const char *file, const char *func, int line);
```

Comment:
Typically WiredTiger macros intentionally leave out the ; so when used they still require one in the actual usage location. I would remove this ;

Reply:
The macro is defined on only new line 16 which does not have a semicolon.

New line 17 is not part of the macro. It is a function declaration for the function called by the macro. The function definition  is in utils.cpp.

I added a blank line between new line 16 and new line 17 to make this more clear.

Reply:
I also moved the #define out of the utils namespace for readability too.